client/client_api.go

```diff
@@ -4,13 +4,18 @@ import (
 	"context"
 
 	"github.com/gogo/protobuf/proto"
+
 	api "github.com/ligato/vpp-agent/api/genericmanager"
 )
 
+type ModelInfo = api.ModelInfo
+
+type StateItem = api.StateItem
```

Comment:
Isn't it redundant creating aliases with the same name as original types?

Reply:
These will be defined as new types later and the API won't have to change for the rest of the code.